content/blog/2025/12/arduino-library-creation/index.md

```diff
@@ -0,0 +1,473 @@
+---
+title: "Creating an Arduino Library for ESP32"
+date: 2025-12-04
+tags: ["Arduino", "ESP32", "Library", "Tutorial"]
+showAuthor: false
+featureAsset: "img/featured/featured-arduino.webp"
```

Comment:
I suggest adding a new featured image. We can work on it if you don't have any other image to use.

Reply:
Yes please. I don't have any other.